python/lsst/ts/eas/eas_csc.py

```diff
@@ -308,6 +347,74 @@ async def run_control(self) -> None:
                 )
                 break
 
+    async def m1m3_stop(self) -> None:
+        """Ends the M1M3 control loop.
+
+        Commands the mixing valve to close and the fans to stop.
+        Waits for telemetry to report that the mixing valve
+        has closed and the fans have stopped.
+        """
+
+        if self.simulation_mode != 0:
```

Comment:
I am a bit uncomfortable with the proliferation of this `if self.simulation_mode != 0` all over the code base. I would suggest you remove these.